ci-operator/step-registry/ipi/conf/ovirt/ipi-conf-ovirt-chain.yaml

```diff
@@ -4,7 +4,6 @@ chain:
   - ref: ipi-conf-ovirt-generate-install-config-params
   - ref: ipi-conf-ovirt-generate-ovirt-config
   - ref: ipi-conf-ovirt-generate-install-config
-  - ref: ipi-install-monitoringpvc
```

Comment:
I'd prefer default chain to have PVC (as we release new versions there'd be only two version without CSI support, rapidly moving to EOL)

Reply:
I prefer not to add it on the ovirt conf chain because I don't want to have multiple ovirt conf chains and multiple pre chains.
I just keep the default ovirt conf chain without a reference to the monitotingpvc (same as the etcd ramfs) and have it contain only the ovirt configuration steps, and reference the monitotingpvc and etcd ramfs steps on the pre chains